packages/datadog_flutter_plugin/README.md

```diff
@@ -168,6 +168,38 @@ final configuration = DdSdkConfiguration(
 
 In order to enable Datadog Distributed Tracing, the `DdSdkConfiguration.firstPartyHosts` property in your configuration object must be set to a domain that supports distributed tracing. You can also modify the sampling rate for Datadog distributed tracing by setting the `tracingSamplingRate` on your `RumConfiguration`.
 
+## Troubleshooting
+
+### Cocoapods issues
+
+If you are having trouble building your iOS application after adding the Datadog SDK because of errors being thrown by Cocoapods, check which error you are getting. The most common error is an issue getting the most up to date native library from Cocoapods, which can be solved by running the following in your `ios` directory:
```

Comment:
```suggestion
If you have trouble building your iOS application after adding the Datadog SDK because of errors being thrown by Cocoapods, check which error you are getting. The most common error is an issue getting the most up-to-date native library from Cocoapods, which can be solved by running the following in your `ios` directory:
```